core/src/test/scala/org/apache/spark/sql/delta/CheckpointsSuite.scala

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.delta
+
+import java.net.URI
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.permission.FsPermission
+import org.apache.hadoop.fs.{FSDataOutputStream, Path, RawLocalFileSystem}
+import org.apache.hadoop.util.Progressable
+import org.apache.spark.sql.QueryTest
+import org.apache.spark.sql.delta.storage.LocalLogStore
+import org.apache.spark.sql.test.SharedSparkSession
+
+class CheckpointsSuite extends QueryTest with SharedSparkSession {
+
+  protected override def sparkConf = {
+    super.sparkConf.set("spark.delta.logStore.gs.impl", classOf[LocalLogStore].getName)
+  }
+
+  ignore("isGCSPath") {
```

Comment:
why is this ignore?

Reply:
Good catch. Forgot to change it back when I was testing other tests.